aries-site/src/pages/components/layer/center-layer.mdx

```diff
@@ -0,0 +1,68 @@
+import { Notification } from 'grommet';
+import { Example } from '../../../layouts';
+import { CenterLayerAnatomy, ContentDrivenLayout } from '../../../examples';
+
+## Anatomy
+
+This diagram highlights details specific to center layers. See [general layer anatomy](/components/layer#anatomy) for specifications applying to all layers.
+
+<Example plain>
+  <CenterLayerAnatomy />
+</Example>
+
+| Label | Region             | Purpose                                                          | Required?                                  | Notes                                                                                                                                                                                                                                |
+| ----- | ------------------ | ---------------------------------------------------------------- | ------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| **1** | **Close button**   | Closes the layer.                                                | Required on layers with no footer actions. | Use "FormClose" icon.                                                                                                                                                                                                                |
```

Comment:
This seems to be in conflict with the guidance given on the general Layer page:
> For actionable layers that require user actions, place a "Cancel" action alongside the other footer actions. In some cases, a label other than "Cancel" may be appropriate.
Do not include an "X" button in the header to avoid confusion of how the user should close the layer.

Reply:
I'm not following the conflict. Can you elaborate?

My understanding is we do footer actions (actionable layer) OR and "X" in the header (informational layer).

Reply:
Maybe it's that the image shows both, and doesn't make it clear right there that it shouldn't look like the example.

![Screenshot 2022-11-22 at 3 01 19 PM](https://user-images.githubusercontent.com/11637956/203438599-42689205-5adb-4e3f-bfe7-7dc6e7541a55.png)

Reply:
I get that, was struggling how to handle that and was hoping people would be reading the table that accompanies for details about when to show them and their purpose. 

What do you recommend to improve-- maybe 2 anatomy diagrams? 1 for informational, 1 for actionable?

Reply:
I think I'm inclined towards two anatomy diagrams, which might help reinforce that the user needs to think about the distinction more, and make the right choice. I've been on prior projects where people just look at an example and replicate the visual without reading the guidance. Then, you have to point them to the guidance and get them to make the adjustment visually in their minds. Just giving them a more straightforward visual to start with prevents this type of behavior. But, soft opinion.